muspinsim/muspinsim.xml

```diff
@@ -61,6 +61,9 @@
             <discover_datasets pattern="__name__" directory="data" format="txt"/>
         </collection>
         <data label="muspinsim log for $mu_sim.name" name="log_out" format="txt" from_work_dir="log_out.log"/>
+        <data label="fit report for $mu_sim.name" name="fit_report" format="txt" from_work_dir="fitting_data_fitreport.txt">
```

Comment:
When I try this tool in the browser with the muspinsim fitting example, the fit_report file comes back empty (0 lines). I don't think this line is finding the file properly. Can you replicate this?

Reply:
my mistake - forgot to take into account the prefix of the file changes based on the keyword `name`